website/docs/guides/orchestration/webhooks/zapier-new-cloud-job.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Trigger another dbt Cloud job when a run finishes"
+id: webhooks-guide-zapier-new-cloud-job
+slug: zapier-new-cloud-job
+---
+
+This guide assumes familiarity with:
```

Comment:
similar suggestion to the comment about:

- moving the h2 'what does this integration do' section to the top, and
- making this 'assumes familiarity' bit as a prerequisite, and moving it down.